rootfs-builder/ubuntu/Dockerfile.in

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2018  Yash Jain
```

Comment:
Nit: double space after year.

Also, can I just confirm the copyright name is correct (should it be the company you work for?)

Reply:
Yup It should be in my name.